vllm/model_executor/layers/quantization/fp8.py

```diff
@@ -229,6 +229,24 @@ def process_weights_after_loading(self, layer: Module) -> None:
                 raise ValueError(
                     f"Unknown scheme {self.quant_config.activation_scheme}")
 
+    def cutlass_fp8_supported(self) -> bool:
```

Comment:
It might be a good idea to not have this on the hotpath and instead query this during initialization